content/influxdb/v2.0/upgrade/v1-to-v2.md

```diff
@@ -12,33 +12,108 @@ aliases:
 ---
 
 Use the `influxd upgrade` command to upgrade InfluxDB 1.x to InfluxDB 2.0.
-This command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
+The `upgrade` command copies all data stored in 1.x [databases](/influxdb/v1.8/concepts/glossary/#database) and
 [retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)
 to 2.0 [buckets](/influxdb/v2.0/reference/glossary/#bucket).
 
+<!-- TODO summarize effects on other resources -->
+
+## Before you begin: important considerations
+
+Before upgrading to InfluxDB 2.0, please consider the following issues;
+some or all might apply to your specific installation and use case.
+The sections below contain our recommendations for addressing possible gaps in the upgrade process.
+You may also find while reading that you'd like to hold off on upgrading for now.
+
+### Continuous queries
+
+When upgrading from InfluxDB 1.x and using continuous queries (CQs) to downsample data,
+we recommend extracting CQs from your InfluxDB 1.x instance prior to upgrade with `show continuous queries`.
+Run the following:
+
+```sh
+$ influx
+Connected to http://localhost:8086 version 1.8.3
+InfluxDB shell version: 1.8.3
+> show continuous queries
+```
+
+Save this output so it can be used later.
+
+Continuous Queries are replaced by Tasks in InfluxDB 2.0
+To convert your CQs to Flux, see the [Tasks documentation](/influxdb/v2.0/process-data/get-started/).
+
+### Supported protocols
+
+If you're using one of the [supported protocols](/influxdb/v1.8/supported_protocols/) in InfluxDB 1.x (CollectD, Graphite, OpenTSDB, Prometheus, UDP),
+you may want to research using Telegraf instead.
+InfluxDB 2.0 doesn't support these protocols directly.
+Use Telegraf to translate from the source protocol to InfluxDB 2.0.
+
+### Kapacitor
```

Comment:
The subscriptions API should reside here. 
---
OSS users can continue to use Kapacitor with InfluxDB OSS 2.0 under the following scenarios:

1.  Kapacitor Batch-style TICKscripts work with the 1.x read compatible API
Existing Kapacitor user credentials should continue to work as they will leverage the  1.x read compatibility API.

2. Kapacitor Stream-style TICKscripts -- will not work using the subscription API (as there isn’t one present in 2.0.
However, best practice is to write data directly to BOTH Kapacitor and InfluxDB.  This allows stream tasks to continue to work.  (See dual writes using Telegraf.) 
If you use other mechanisms to feed data in, Telegraf could be used as an interstiatial layer to feed both Kapacitor and InfluxDB.  There are other architectures and mechanisms possible.